transformer_engine/common/recipe/__init__.py

```diff
@@ -77,6 +77,10 @@ def float8_current_scaling(self):
         """Whether the given recipe is (per-tensor) current scaling."""
         return isinstance(self, Float8CurrentScaling)
 
+    def per_tensor_scaling(self):
```

Comment:
We are using this to check FP8 in addition to per-tensor scaling, so the name should reflect this:
```suggestion
    def float8_per_tensor_scaling(self):
```

Reply:
Thanks for pointing out, have made the corresponding changes.